test/colorado_lottery_test.rb

```diff
@@ -0,0 +1,91 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require "./lib/contestant"
+require "./lib/game"
+require "./lib/colorado_lottery"
+
+
+class ColoradoLotteryTest < Minitest::Test
+  def setup
+    @lottery = ColoradoLottery.new
```

Comment:
I like that you used the `setup` method here. You could have also added each contestants interests here.